RunTestSuite.py

```diff
@@ -29,9 +31,8 @@ def getTestResults(output):
 
 
 def runAndroidTest():
-    subprocess.call("gradle clean", shell=True)
```

Comment:
I had to remove this because this would cause the different stress test processes to interfere with each other by changing the build output.